File: src/Backend/Domain/Auth.Entity/Ids4Entity/ClientClaim.cs
using System;
using Dapper.Contrib.Plus;

namespace Auth.Entity.Ids4Entity
{
    /// <summary>
    /// ClientClaim
    /// </summary>
    [Table("client_claims")]
    public class ClientClaim
    {
        /// <summary>
        /// 主键
        /// </summary>
        [Key]
        public int Id { get; set; }

        /// <summary>
        /// Type
        /// </summary>
        public string Type { get; set; }

        /// <summary>
        /// Value
        /// </summary>
        public string Value { get; set; }

        /// <summary>
        /// ClientId
        /// </summary>
        public int ClientId { get; set; }
    }
}

File: src/FrontEnd/auth-admin/src/types/ids/client-claim.ts
interface IClientClaim {
  id?: number;
  type: string;
  value: string;
  clientId: number;
}

export default class ClientClaim implements IClientClaim {
  id?: number | undefined;
  type: string = '';
  value: string = '';
  clientId: number = 0;
}